roles/jd-server/src/lib/job_declarator/message_handler.rs

```diff
@@ -38,6 +41,41 @@ impl JobDeclaratorDownstream {
     }
 }
 
+pub fn clear_old_mempool_transactions(
```

Comment:
I am a bit on the fence with this name. The struct is `DeclareMiningJob`  so maybe it would make more sense to change this to `fn clear_old_mining_jobs`. Thinking about it, why not adding this function in a 
```rust
impl DeclareMiningJob { 
  pub(crate) fn clear_from_mempool(&self, mempool: Arc<Mutex<..>>) {} 
}
```

Reply:
I am not clearing old mining jobs in this case just emptying the mempool from the old mining jobs

Reply:
> I am not clearing old mining jobs in this case just emptying the mempool from the old mining jobs

you are clearing the txs of old mining jobs from the jds light mempool.
What @jbesraa  wrote make sense to me

Reply:
Alright, will adjust that now